src/region/RegionAnnotations.tsx

```diff
@@ -74,13 +74,14 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
     };
 
     handleSubmit = (): void => {
-        const { createRegion, message, staged } = this.props;
+        const { createRegion, message, setMessage, staged } = this.props;
 
         if (!staged) {
             return;
         }
 
         createRegion({ ...staged, message });
+        setMessage('');
```

Comment:
Do we need to `setStaged(null)` as well? Should we have a `reset` helper method?